modules/ROOT/pages/_partials/keycloak/setting-up-the-idm-sdk.adoc

```diff
@@ -70,20 +70,20 @@ Cordova::
 +
 --
 // end::excludeDownstream[]
-. Add `* file:` to `default-src` in the `Content-Security-Policy` meta tag inside the `index.html` file:
+. Add the following code to `index.html` located in `src/` :
 +
 [source,html]
 ----
 <meta http-equiv="Content-Security-Policy" content="default-src * file: 'self' data: gap: https://ssl.gstatic.com 'unsafe-eval'; style-src 'self' 'unsafe-inline'; media-src *; img-src 'self' data: content:;">
 ----
-. Install the link:https://www.npmjs.com/package/@aerogear/auth[AeroGear Auth] package from link:https://www.npmjs.com/[NPM^]:
+. Install the link:https://www.npmjs.com/package/@aerogear/auth[AeroGear Auth] package from link:https://www.npmjs.com/[NPM^] in `src/`:
 +
 [source,bash]
 ----
 $ npm install @aerogear/auth
 ----
-. Use e.g `browserify` so that you can use the `require` statement
-. In the `init` snippet use `require` instead of `import`
+. In order for a node module to work in a browser (and on Cordova), use `browserify` to allow for the use of a `require` statement.
```

Comment:
I thin this might need to be addressed in a concept topic: "Introducing JavaScript bundling"

Reply:
I'm thinking that this relates to the comment above and should be part of a Content Strategist piece of work that is planned for GA.

Reply:
browserify is and example of a JavaScript blunder but it is not the only one out there https://www.slant.co/topics/3900/~best-frontend-javascript-module-bundlers. We shouldn't be mentioning browsers as this is a Cordova implementation  
```suggestion
Use a JavaScript blunder e.g `browserify` so that you can use node modules in Cordova. We recommend the `require` statement rather that `import` for the use of the @aerogear/auth module as outline in the code snippet below.
```